jbmc/regression/jbmc/exception-cleanup/test.desc

```diff
@@ -0,0 +1,14 @@
+CORE
+Test.class
+--function Test.main
+^EXIT=0$
+^SIGNAL=0$
+^VERIFICATION SUCCESSFUL$
+1 remaining after simplification$
+--
+^warning: ignoring
+--
+The unreachable function should be successfully marked unreachable by symex,
+but the final uncaught exception is not yet, as it requires symex to note that
```

Comment:
I don't understand what "the final uncaught exception" means. All the exceptions appear to be caught to me.

Reply:
It refers to the "did any exception escape at top level?" test that's always generated in `__CPROVER_start`, and yes they are all caught, but symex isn't *quite* smart enough to determine that yet. This PR enables it to determine there's no exception in flight when we return to normal code (preventing exploration of catch blocks after future no-throw function calls), but doesn't yet enable it to determine catch block does not rethrow.

Reply:
(I'll improve the wording here)